constants/chapter.json

```diff
@@ -47,9 +47,43 @@
     "title": "Chapter 3",
     "subtitle": "Functions",
     "image": "/chapter-3.jpg",
-    "exercises": "",
+    "exercises": "Minimum, Recursion, Bean counting",
     "link": "/chapter3",
-    "details": ""
+    "details": "This chapter taught you how to write your own functions. Separating the tasks your program performs into different functions is helpful. You won’t have to repeat yourself as much, and functions can help organize a program by grouping code into pieces that do specific things.",
+    "devOscar": [
+      {
+        "id": 1,
```

Comment:
Feels like we don't need these `id`s since the title is already unique and can be used as an id?

Reply:
I am using this id to link with every page, if I remove it, it will crash

Reply:
Ok